index.html

```diff
@@ -48,8 +50,166 @@ <h3 class="heading-3">Let's connect</h3>
       <a href="#">
         <img class="social-icons" src="./images/fingure-img.png" alt="fingure icon">
       </a>
+      </div>
     </section>
+    <!-----===== Work section ======---->
 </section>
-  </header>
+<main> 
+<div class="card-container">
```

Comment:
- [ ] Kindly consider taking a `section` tag here as a container for the works section. it is due to following best practices, semantics and convention.